docarray/documents/audio.py

```diff
@@ -94,11 +96,27 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[AudioUrl]
-    tensor: Optional[AudioTensor]
-    embedding: Optional[AnyEmbedding]
-    bytes_: Optional[AudioBytes]
-    frame_rate: Optional[int]
+    url: Optional[AudioUrl] = Field(
+        description='The url of the audio file.',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/hello.mp3?raw=true',
+    )
+    tensor: Optional[AudioTensor] = Field(
+        description='''Tensor object of the audio which be specified to one of
+        `AudioNdArray`, `AudioTorchTensor`, `AudioTensorFlowTensor`''',
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='''Embedding field is used to store tensor objects of type
+        Tensorflow, PyTorch, NumPy and Jax''',
+        example='np.zeros((3, 32, 32))',
+    )
+    bytes_: Optional[AudioBytes] = Field(
```

Comment:
Same, skipping example here since it can only be loaded from the url by giving a multiline example